cluster/azure/pod_identity/deploy_pod_identity.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# parse command-line arguments
+while getopts :u: option 
+do 
+ case "${option}" in 
+ u) POD_IDENTITY_DEPLOYMENT_URL=${OPTARG};;
+ *) echo "ERROR: Please refer to usage guide on GitHub" >&2
+    exit 1 ;;
+ esac
+done 
+
+# Deploy the flex volume support into the cluster
```

Comment:
Perhaps "Deploy the pod identity support..." is intended in the comment instead of "flex volume".